manifests/baremetal/keepalived.yaml

```diff
@@ -52,20 +52,31 @@ spec:
             /usr/sbin/keepalived -f /etc/keepalived/keepalived.conf --dont-fork --vrrp --log-detail --log-console &
         fi
       }
+      stop_keepalived()
+      {
+        echo "Keepalived process stopped" >> /etc/keepalived/stopped
```

Comment:
Not a big deal, but since the control socket is in /var/run/keepalived, maybe we put the stop file there too?